### PR TITLE
Add forgotten stories of football Acast group

### PR DESCRIPTION
**Pull request by JustinPinner**
Adds a tag in preparation for Acast setting up the new podcast.

### CODE REVIEW

**Comment by Mullefa** [comment]
I've been assigned? What does that mean? 😆

**Comment by JustinPinner** [comment]
Ha sorry @Mullefa @juanvaccari I meant to put you in as reviewers, not assignees 🙄

File: app/com/gu/itunes/iTunesRssItem.scala [hunk 1]
```diff
@@ -119,7 +119,9 @@ class iTunesRssItem(val podcast: Content, val tagId: String, asset: Asset) {
         AcastLaunchGroup(new DateTime(2019, 10, 7, 0, 0), Seq(
           "australia-news/series/full-story")),
         AcastLaunchGroup(new DateTime(2020, 1, 28, 0, 0), Seq(
-          "science/series/thegenegapcommonthreads")))
+          "science/series/thegenegapcommonthreads")),
+        AcastLaunchGroup(new DateTime(2020, 5, 7, 0, 0), Seq(
```
Comment (Mullefa) [inline app/com/gu/itunes/iTunesRssItem.scala:123]: Out of interest, did you infer the date and tag from the urls - https://www.theguardian.com/sport/audio/2020/may/07/forgotten-stories-of-football-podcast-season-one-trailer
and https://www.theguardian.com/football/series/forgotten-stories-of-football - referenced in the email (respectively) ?

Comment (JustinPinner) [reply app/com/gu/itunes/iTunesRssItem.scala:123]: That date is used to indicate when the podcast went live. In this case as it is live from today I just used that date.

Comment (Mullefa) [reply app/com/gu/itunes/iTunesRssItem.scala:123]: Thanks. And the tag you inferred from the url?

Comment (JustinPinner) [reply app/com/gu/itunes/iTunesRssItem.scala:123]: Exactly that, yes. The URL is a tag page (one of our favourite kinds of page!)

**Comment by JustinPinner** [comment]
We've got the green light from Acast 🚦 so I'm going to remove the Do Not Merge tag and get this merged now...

**Pull request merged**